docs/Community/governance/growth-plan.md

```diff
@@ -0,0 +1,51 @@
+# Growth Plan
+
+## Introduction
+
+The Open Enclave SDK is an
+[Incubation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#incubation-stage)
+project of the Confidential Computing Consortium (CCC), with an approved
+[Technical Charter](/TECHNICAL_CHARTER.md).
+
+One requirement for Incubation stage projects is to develop, in
+conjunction with the mentor assigned by the CCC Technical Advisory
+Council, a growth plan for reaching the
+[Graduation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#graduation-stage)
+over time.
+
+## Graduation Stage Acceptance Criteria
+
+This section tracks the progress towards the Acceptance Criteria for the
+Graduation Stage, as defined by the CCC:
+
+- [ ] Have a defined [governing body](README.md) of at least 5 or more members
+(owners and core maintainers, or similar technical role), of which no
+more than 1/3 is affiliated with the same employer. In the case there
+are 5 governing members, 2 may be from the same employer.
+
+- [ ] Have a healthy number of [committers](../Committers.md#list-of-committers) from at least two organizations.
```

Comment:
Is there a specific on what number is considered healthy?

Reply:
Nope, there's no objective criteria at this point.   The CCC might come up with sometihng in the future, but none currently.

Reply:
We can look towards other organizations, such as the CNCF or OpenStack, for examples of criteria that may be adopted at the CCC level. Such examples are merely illustrative and the CCC will eventually make its own criteria. 

As an example from OpenStack, the bar used to include a check that the most active company has less than one third of the active contributors in the last 6 months for evaluation of project-member diversity.